### PR TITLE
refine and verify ipex-llm-serving-xpu docker document

### PR DESCRIPTION
**Pull request by liu-shaojun**
### 1. Why the change?

This submission refines and verifies ipex-llm-serving-xpu docker document

### CODE REVIEW

**Review by gc-fu — APPROVED** [review]
LGTM

File: docker/llm/README.md [hunk 1]
```diff
@@ -229,102 +236,131 @@ docker run -itd \
...
+  python3 -m ipex_llm.serving.fastchat.ipex_llm_worker --model-path path/to/vicuna-7b-v1.5 --low-bit "sym_int4" --trust-remote-code --device "cpu" &
+  ```
+  Wait until the process finishes loading the model and you see "Uvicorn running on ...". The model worker will register itself to the controller.
 
-##### Launch the Controller
-```bash
-python3 -m fastchat.serve.controller
-```
+- #### **Step 3: Launch Gradio web server or RESTful API server**
+  You can launch Gradio web server to serve your models using the web UI or launch RESTful API server to serve with cURL.
```
Comment (glorysdj) [inline docker/llm/README.md:267]: cURL => HTTP.

File: docker/llm/README.md [hunk 2]
```diff
@@ -229,102 +236,131 @@ docker run -itd \
...
+  - **Option 2: Serving with OpenAI-Compatible RESTful APIs**
 
-##### Launch the Gradio web server
+      Launch the RESTful API server
 
-```bash
-python3 -m fastchat.serve.gradio_web_server
-```
+      ```bash
+      python3 -m fastchat.serve.openai_api_server --host localhost --port 8000 &
```
Comment (glorysdj) [inline docker/llm/README.md:282]: localhost to 0.0.0.0? and please test serving with ip address.

File: docker/llm/README.md [hunk 3]
```diff
@@ -229,102 +236,131 @@ docker run -itd \
...
-By following these steps, you will be able to serve your models using the web UI with `IPEX-LLM` as the backend. You can open your browser and chat with a model now.
+      ```bash
+      curl -X POST -H "Content-Type: application/json" -d '{
+        "model": "Llama-2-7b-chat-hf",
+        "prompt": "Once upon a time, there existed a little girl who liked to have adventures. She wanted to go to places and meet new people, and have fun",
+        "n": 1,
+        "best_of": 1,
+        "use_beam_search": false,
+        "stream": false
+      }' http://localhost:8000/v1/completions
```
Comment (glorysdj) [inline docker/llm/README.md:295]: localhost => YOUR_HTTP_HOST

File: docker/llm/README.md [hunk 4]
```diff
@@ -229,102 +236,131 @@ docker run -itd \
...
 
-To start an OpenAI API server that provides compatible APIs using `IPEX-LLM` backend, you need three main components: an OpenAI API Server that serves the in-coming requests, model workers that host one or more models, and a controller to coordinate the web server and model workers.
+- #### **Step 1: Launch the api_server**
+  ```bash
+  #!/bin/bash
+  # You may also want to adjust the `--max-num-batched-tokens` argument, it indicates the hard limit
+  # of batched prompt length the server will accept
+  numactl -C 0-47 -m 0 python -m ipex_llm.vllm.entrypoints.openai.api_server \
+          --model /llm/models/Llama-2-7b-chat-hf/ --port 8000  \
+          --load-format 'auto' --device cpu --dtype bfloat16 \
```
Comment (glorysdj) [inline docker/llm/README.md:309]: please check if it can support 0.0.0.0

File: docker/llm/README.md (hunk 1 above)
Comment (liu-shaojun) [reply docker/llm/README.md]: updated

File: docker/llm/README.md (hunk 2 above)
Comment (liu-shaojun) [reply docker/llm/README.md]: tested and updated, serving with ip address works.

File: docker/llm/README.md (hunk 3 above)
Comment (liu-shaojun) [reply docker/llm/README.md]: updated

File: docker/llm/README.md (hunk 4 above)
Comment (liu-shaojun) [reply docker/llm/README.md:310]: tested and updated

**Review by glorysdj — APPROVED** [review]
LGTM

**Pull request merged**